reference/customers.v2.yml

```diff
@@ -1220,32 +1220,107 @@ components:
         category_access:
           $ref: '#/components/schemas/categoryAccessLevel_Full'
         discount_rules:
-          type: array
-          description: A collection of discount rules that are automatically applied to customers who are members of the group.
-          items:
-            type: object
-            properties:
-              type:
-                type: string
-                enum:
-                  - price_list
-                  - all
-                  - category
-                  - product
-              method:
-                type: string
-                enum:
-                  - percent
-                  - fixed
-                  - price
-              amount:
-                type: string
-                description: A float that specifies the value applied to the price modified. (Float, Float as String, Integer)
-                example: "5.0000"
-              price_list_id:
-                type: integer
-                description: If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.
-                example: 3
+          oneOf:
+          - type: array
+            description: A collection of discount rules that are automatically applied to customers who are members of the group.
+            items:
+              type: object
+              properties:
+                type:
+                  type: string
+                  enum:
+                    - price_list
+                    - all
+                    - category
+                    - product
```

Comment:
Should we remove `product` and `category` here because these two cases are covered by the below? I guess if one does not provide `product_id` or `category_id`, then even putting product or category as type here, it wouldn't work 🤔